people/benefits.md

```diff
@@ -56,6 +56,10 @@ We encourage you to attend conferences to improve your knowledge of a subject ar
 
 All expenses for the conference, such as transport, accommodation, fees, food, etc., are covered by your [allowance](#productivity-allowance) and the additional conference allowance.
 
+### Conference fee
+
+The conference free will be reimbursed as a business expense so is seperate from your allowances. If the conference fee is over 500 EUR then it will need to be confirmed by a Partner.
```

Comment:
`so is seperate from your allowances.` to `so it is seperate from your allowances.`